precompile/erc20/permit.go

```diff
@@ -278,12 +266,14 @@ func buildDigest(owner, spender common.Address, amount, nonce, deadline *big.Int
 // and produces the same result as the Solidity code seen e.g. in the OpenZeppelin
 // lib https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/cryptography/EIP712.sol#L89
 // that is used by tBTC token https://github.com/keep-network/tbtc-v2/blob/main/solidity/contracts/token/TBTC.sol#L8
```

Comment:
When it was just BTC precompile comparing it to tBTC kinda made sense. But since we're abstracting this code, not sure if referrencing tBTC is necessary. Up to you to decide, just pointing this out.

Reply:
Yeah I decided to leave it as it's still a useful example of the implementation.